server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -152,7 +152,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongAsyncCounter(
-                "es.node.stats.indices.translog.earliest_last_modified_age",
+                "es.node.stats.indices.translog.earliest_last_modified_age.time",
```

Comment:
Nit: isn't "age" redundant? earliest_last_modified.time looks better, IMO

Reply:
fair point, I suspect this was meant to replicate already existing metrics returned from that node-stats api
I will rename.